l1-contracts/test/delegation/minimal.t.sol

```diff
@@ -60,14 +60,14 @@ contract MinimalDelegationTest is GSEBase {
     Timestamps memory ts;
 
     ts.ts1 = block.timestamp;
-    ts.ts2 = ts.ts1 + 10;
-    ts.ts3 = ts.ts2 + 10;
-
-    require(votingTime > ts.ts3, "ts");
-    ts.ts4 = votingTime + 10;
```

Comment:
Not sure if it will really end up checking the same thing when jumps are this long 🤔 (but test was not good enough if it did not break then). 

As I remember some of the reasoning around 10 was that multiple of these things happened within the voting duration which I think is defaulting to be teeny tiny atm.

Would consider it fine, as this is exactly why there is a bigger issue #14804 to make proper test for it.